src/main/java/dev/andrylat/vomelianchuk/finances/consoleutils/ConsoleIO.java

```diff
@@ -0,0 +1,7 @@
+package dev.andrylat.vomelianchuk.finances.consoleutils;
+
+public interface ConsoleIO {
+    Object read();
```

Comment:
very generic. You can use here generics or interface.

Reply:
Fixed here: ee85a61